docs/DockerlessCalicoManual.md

```diff
@@ -0,0 +1,96 @@
+# Dockerless Calico - Manual Installation
+Project Calico releases are primarily distributed as docker containers for quick, easy, and consistant deployment. However, it is possible to run the core calico components directly on the host, removing the dependency on docker.
+
+This guide will walk through how to manually create services and configurations to run Calico on Centos7 without Docker.
+
+## WARNING: Dockerless Calico is Experimental!
+Some `calicoctl` commands rely on a running `calico-node` container, and may not function properly when calico is run directly on the host. Please raise any encountered issues, or message us on [calico-slack](https://calicousers-slackin.herokuapp.com/).
+
+## Installation
+1. Make changes to SELinux and QEMU config to allow VM interfaces with type='ethernet'. [this libvirt Wiki page][libvirt-wiki] explains why these changes are required):
+    ```
+    setenforce permissive
+    ```
+    
+2. Edit `/etc/selinux/config` and change the `SELINUX=` line to the following:
+    ```
+    SELINUX=permissive
+    ```
+
+3. Add calico's repositories:
+    ```
+    cat > /etc/yum.repos.d/calico.repo <<EOF
+    [calico]
+    name=Calico Repository
+    baseurl=http://binaries.projectcalico.org/rpm_stable/
+    enabled=1
+    skip_if_unavailable=0
+    gpgcheck=1
+    gpgkey=http://binaries.projectcalico.org/rpm/key
+    priority=97
+    EOF
+    ```
+
+4. Calico depends on a few EPEL packages, so be sure you have added the EPEL repos: 
+    ```
+    $ yum install -y epel-release
+    ```
+
+5. Then install calico-felix: 
+    ```
+    $ yum install -y calico-felix
+    ```
+
+6. Until calico-felix 0.3.0 is released, you'll also have to manually install a few runtime deps: 
+    ```
+    $ yum install -y posix-spawn python-gevent python-eventlet python-etcd
+    ```
+
+7. Install the additional binaries for dockerless-calico:
+    ```
+    # calicoctl
+    $ curl -L https://github.com/projectcalico/calico-docker/releases/download/v0.12.0/calicoctl -o /usr/local/bin/calicoctl
+    $ chmod +x /usr/local/bin/calicoctl
+    
+    # bird
+    $ curl -L https://github.com/projectcalico/calico-bird/releases/download/v0.1.0/bird -o /usr/local/bin/bird
+    $ chmod +x /usr/local/bin/bird
+    
+    # confd
+    $ curl -L https://github.com/projectcalico/confd/releases/download/v0.10.0-scale/confd.static -o /usr/local/bin/confd
+    $ chmod +x /usr/local/bin/confd
+    ```
+
+8. Install the required systemd services and confd templates:
+    ```
+    $ git clone https://github.com/project-calico/calico-docker.git
```

Comment:
Do you want to pin this to a specific version of calico-docker so that the guide is independent of the master version of these files?

You'd need to get the files into a release first, but then you can make sure this guide always works.